tabfilter.py

```diff
@@ -40,11 +41,16 @@ def run(self):
 
 		tabs = [entity.get_details(self.prefix, include_path, show_captions) for entity in tabs]
 
+		on_highlight_cb = None
+		selected_index = None
```

Comment:
Ahh, sorry, this was a typo on my part in the last comment - `selected_index` has to be an integer, so we need to default this to `-1`, not `None`, otherwise the quick info window won't show at all when working with multiple panes.